https://swiftshader-review.googlesource.com/c/SwiftShader/+/71268

From 3ecab9c1aa60d548d8efeae3a231f4df7cca6cc7 Mon Sep 17 00:00:00 2001
From: Stephan Hartmann <stha09@googlemail.com>
Date: Sun, 02 Apr 2023 20:53:20 +0200
Subject: [PATCH] IWYU: add cstdint for uint64_t in sw::LRUCache

Bug: chromium:957519
Change-Id: I70970ceda50dfc38f3d149fea03e8a6a79a35934
Reviewed-on: https://swiftshader-review.googlesource.com/c/SwiftShader/+/71268
Commit-Queue: Shahbaz Youssefi <syoussefi@google.com>
Reviewed-by: Shahbaz Youssefi <syoussefi@google.com>
Kokoro-Result: kokoro <noreply+kokoro@google.com>
Tested-by: Shahbaz Youssefi <syoussefi@google.com>
--- a/third_party/swiftshader/src/System/LRUCache.hpp
+++ b/third_party/swiftshade/rsrc/System/LRUCache.hpp
@@ -18,6 +18,7 @@
 #include "System/Debug.hpp"
 
 #include <cstddef>
+#include <cstdint>
 #include <functional>
 #include <unordered_set>
 #include <vector>

